### PR TITLE
[docs] append upstream documentation to project

### PR DESCRIPTION
**Pull request by tobrun**
## Pull request checklist:
 - [x] Briefly describe the changes in this PR.
 - [x] Include before/after visuals or gifs if this PR includes visual changes.
 - [ ] Write tests for all new functionality. If tests were not written, please explain why.
 - [ ] Optimize code for java consumption (`@JvmOverloads`, `@file:JvmName`, etc).
 - [ ] Add example if relevant.
 - [ ] Document any changes to public APIs.
 - [x] Apply changelog label ('breaking change', 'bug :beetle:', 'build', 'docs', 'feature :green_apple:', 'performance :zap:', 'testing :100:') or use the label 'skip changelog'
 - [x] Add an entry inside this element for inclusion in the `mapbox-maps-android` changelog: `<changelog>Append gl-native and common API reference documentation to the output of Dokka documentation generation.</changelog>`.

#### Before
Screenshot from 2021-10-04 18-43-28

#### After
Screenshot from 2021-10-04 18-22-40

### Notes

- need to update documentation and scripts to publish documentation
- fine tune dokka configuration to replace the title in screenshot to `mapbox-maps-android`

### CODE REVIEW

**Review by axti — APPROVED** [review]

**Review by ank27 — APPROVED** [review]

**Comment by kiryldz** [comment]
@tobrun we need also update instructions, perhaps README here, def we need to update release template.

File: scripts/update-android-docs.sh [hunk 1]
```diff
@@ -59,7 +59,7 @@ fi
 
 #Split version name to remove the "android-v" prefix
 array=(`echo $MAPS_SDK_VERSION | tr 'v' ' '` )
-MAPS_SDK_VERSION=${array[1]}
+MAPS_SDK_VERSION=${array[0]}
```
Comment (kiryldz) [inline scripts/update-android-docs.sh:62]: basically we store version as `VERSION_NAME=10.0.0` - guess we can get rid of array completely here

File: scripts/update-android-docs.sh [hunk 2]
```diff
@@ -145,7 +145,6 @@ function create_pull_request() {
 gh auth login --with-token < gh_token.txt
 
 # Generate docs, create branch and make PR with API documentation in the SDK repo.
-generate_docs
```
Comment (kiryldz) [inline scripts/update-android-docs.sh:148]: can we remove associated function from bash script as well?

Comment (tobrun) [reply scripts/update-android-docs.sh:148]: :+1:

**Review by kiryldz — APPROVED** [review]

**Pull request merged**